app/services/return-requirements/submit-points.service.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+/**
+ * Orchestrates validating the data for `/return-requirements/{sessionId}/points` page
+ * @module SubmitPointsService
+ */
+
+const FetchPointsService = require('../../services/return-requirements/fetch-points.service.js')
+const PointValidation = require('../../validators/return-requirements/point.validator.js')
+const SelectPointsPresenter = require('../../presenters/return-requirements/points.presenter.js')
+const SessionModel = require('../../models/session.model.js')
+
+/**
+ * Orchestrates validating the data for `/return-requirements/{sessionId}/points` page
+ *
+ * It first retrieves the session instance for the returns requirements journey in progress.
+ *
+ * The user input is then validated and the result is then combined with the output of the presenter to generate the page data needed by the view.
+ * If there was a validation error the controller will re-render the page so needs this information. If all is well the
+ * controller will redirect to the next page in the journey.
```

Comment:
Looks like some of the lines have gone over our 120 char line limit. Could also do with a bit of a tidy! 😁 

```suggestion
 * Orchestrates validating the data for `/return-requirements/{sessionId}/points` page
 *
 * It first retrieves the session instance for the returns requirements journey in progress.
 *
 * The user input is then validated and the result is then combined with the output of the presenter to generate the
 * page data needed by the view. If there was a validation error the controller will re-render the page so needs this
 * information. If all is well the controller will redirect to the next page in the journey.
```

I highly recommend the VSCode extension [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) to keep a handle on comments.